src/always-be-playing.js

```diff
@@ -65,11 +57,15 @@ export default class GapSkipper {
   }
 
   /**
-   * Periodically check for timeupdates to see if a gap has been encountered.
+   * Periodically check for timeupdates
```

Comment:
Might be better to say "Periodically check the currentTime" since we are no longer using timeupdate events